docs/DeveloperGuide.md

```diff
@@ -492,8 +522,38 @@ The following activity diagram shows the logic of the `help` command.
 
 **Overview:**
 
+The `delete-t` command is used to delete individual tags from an existing `Person` in ConnectUS specified by the user.
+
+The format for the `delete-t` command can be found [here](https://ay2223s2-cs2103t-w15-1.github.io/tp/UserGuide.html#47-deleting-tags-from-a-contact-delete-t).
+
+<div style="page-break-after: always"></div>
+
 **Feature Details:**
 
+1. The user specifies a person index that represents a `Person` to be edited, followed by the tag index to be deleted.
+2. If a negative or zero index is provided, an error is thrown. The user is prompted to re-enter the command correctly.
+3. If the index is not in valid range of the contact list provided, an error is thrown. The user is prompted to re-enter the command correctly.
```

Comment:
Would it be good to add implementation details on this code from DeleteTagFromPersonCommand as well? 

```java
if (moduleIndex != null) {
            var originalModules = convertSetToList(personToEdit.getModules());
            if (!isIndexValid(moduleIndex, originalModules)) {
                throw new CommandException(String.format(Messages.MESSAGE_INVALID_DISPLAYED_INDEX, "module"));
            }
            editedModules = createEditedTagList(originalModules, moduleIndex);
        }

        if (ccaIndex != null) {
            var originalCcas = convertSetToList(personToEdit.getCcas());
            if (!isIndexValid(ccaIndex, originalCcas)) {
                throw new CommandException(String.format(Messages.MESSAGE_INVALID_DISPLAYED_INDEX, "CCA"));
            }
            editedCcas = createEditedTagList(originalCcas, ccaIndex);
        }

        if (majorIndex != null) {
            var originalMajors = convertSetToList(personToEdit.getMajors());
            if (!isIndexValid(majorIndex, originalMajors)) {
                throw new CommandException(String.format(Messages.MESSAGE_INVALID_DISPLAYED_INDEX, "major"));
            }
            editedMajors = createEditedTagList(originalMajors, majorIndex);
        }

        if (remarkIndex != null) {
            var originalRemarks = convertSetToList(personToEdit.getRemarks());
            if (!isIndexValid(remarkIndex, originalRemarks)) {
                throw new CommandException(String.format(Messages.MESSAGE_INVALID_DISPLAYED_INDEX, "remark"));
            }
            editedRemarks = createEditedTagList(originalRemarks, remarkIndex);
        }
```